src/types/Movie.ts

```diff
@@ -0,0 +1,7 @@
+export interface Movie {
```

Comment:
The code provided does not seem to be related to the task description. Please provide the correct code for the MoviesList component that implements the filtering using useState hook.